src/_pytest/stepwise.py

```diff
@@ -8,53 +9,70 @@
 from _pytest.main import Session
 from _pytest.reports import TestReport
 
+if TYPE_CHECKING:
+    from _pytest.cacheprovider import Cache
+
+STEPWISE_CACHE_DIR = "cache/stepwise"
+NO_PREVIOUSLY_FAILED_WONT_SKIP = "no previously failed tests, not skipping."
```

Comment:
For this and the next two constants, I think it is better to keep it as before. The strings are only used once, so I think the code is easier to read when they are just written where they are used.